src/wallet/ismine.h

```diff
@@ -17,13 +17,13 @@ class CScript;
 /** IsMine() return codes */
 enum isminetype : unsigned int
 {
-    ISMINE_NO         = 0,
-    ISMINE_WATCH_ONLY = 1 << 0,
-    ISMINE_SPENDABLE  = 1 << 1,
-    ISMINE_USED       = 1 << 2,
-    ISMINE_ALL        = ISMINE_WATCH_ONLY | ISMINE_SPENDABLE,
-    ISMINE_ALL_USED   = ISMINE_ALL | ISMINE_USED,
-    ISMINE_ENUM_ELEMENTS,
+    ISMINE_NO         = 0,                                    //!< the scriptPubKey is not contained in the wallet
+    ISMINE_WATCH_ONLY = 1 << 0,                               //!< the scriptPubKey has been imported into the wallet
+    ISMINE_SPENDABLE  = 1 << 1,                               //!< the scriptPubKey is corresponding to an address owned by the wallet user (can spend with the private key)
+    ISMINE_USED       = 1 << 2,                               //!< the scriptPubKey is corresponding to an reused address owned by the wallet user (can spend with the private key)
```

Comment:
used, not *re-*used, after all this is part of the avoid-reuse logic :smile:

Reply:
also "(can spend with the private key)" doesn't belong here, `USED` doesn't imply anything about spendability in the future